articleList.scss

```diff
@@ -0,0 +1,36 @@
+@import "../variables";
+
+.insights-articleList {
+    @include transform(translateY(10px));
+    opacity: 100;
```

Comment:
opacity:100 it's not a mistake, but it would be easier to use opacity: 1. Because every value above 1, it' s fully opaque